packages/trackerless-network/src/logic/neighbor-discovery/NeighborUpdateManager.ts

```diff
@@ -16,6 +16,7 @@ interface NeighborUpdateManagerConfig {
     streamPartId: StreamPartID
     rpcCommunicator: ListeningRpcCommunicator
     neighborUpdateInterval: number
+    neighborCount: number
```

Comment:
Is this the preferred neighbor count? I.e. the number of neigbors we'd like to have, but not necessarily currently have?

If that's the case, could rename e.g. to `preferredNeighborCount`

Reply:
Could be misleading as well, e.g. "what is a preferred neighbor and how is that different from normal neighbor?"

I would've suggested `neighborTargetCount`?

Reply:
Should probably rename the variable across all uses then

Reply:
I see what you mean.

The `neighborCount` config here is passed all the way down from the `StreamrNodeConfig#streamPartitionNeighborCount`. We would have to rename all the places (for consistency) and also change the packages/client/src/config.schema.json file.

To keep the PR compact and to the point, I would propose we keep the name as-is (for consistency) and add to future improvements section the name change and land that as a separate refactor PR later. Thoughts?

Reply:
agree

Reply:
Created ticket NET-1257 on this.